src/components/TimelineCard.js

```diff
@@ -11,14 +11,14 @@ export const TimelineCardWrapper = React.forwardRef((props, ref) => {
 
 const type = {commit: 'commit', deploy: 'deploy'};
 
-type TimelineCardProps = {
-    timeline: any;
-    includeRepo: boolean;
-    ghURL: string;
-    glURL: string;
-}
+// type TimelineCardProps = {
+//     timeline: any;
+//     includeRepo: boolean;
+//     ghURL: string;
+//     glURL: string;
+// }
```

Comment:
Do we no longer need the type checking for includeRepo, ghURL, and glURL here?

Reply:
They never _needed_ to be checked, but I liked how explicit it was. The data comes from the api and then is mapped to each card, so I think its pretty safe.

The babel-loader though does not recognize type definitions, so I removed it and 'cast' it on line 22. I wish there was a way to make it less picky.

Reply:
Sounds good!